### PR TITLE
Tag Xpress.jl v0.3.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaOpt/Xpress.jl](https://github.com/JuliaOpt/Xpress.jl)
Release: [v0.3.0](https://github.com/JuliaOpt/Xpress.jl/releases/tag/v0.3.0)
Diff: [vs v0.2.1](https://github.com/JuliaOpt/Xpress.jl/compare/6fe41930a5917b85c5c48309258f43829f77e55d...faaf07efd0b278d4144a5b27bae9ed8899c9ec9a)
`requires` vs v0.2.1: no changes
cc: @joaquimg

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: Xpress/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+MathProgBase 0.5 0.7
+Compat 0.8.6
```
Comment (tkelman) [inline Xpress/versions/0.3.0/requires]: need 0.18 for `@compat const Bounds{T<:Real}`

File: Xpress/versions/0.3.0/requires [hunk 2]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
```
Comment (tkelman) [inline Xpress/versions/0.3.0/requires]: `.&` does not work on 0.4, are you still testing there?

**Comment by joaquimg** [comment]
not testing anymore...

**Comment by tkelman** [comment]
should change the minimum julia version in REQUIRE to reflect the oldest you're testing on

**Comment by joaquimg** [comment]
Ok! Thanks

Joaquim

**Comment by tkelman** [comment]
you re-tagged the same commit without any changea?

**Comment by joaquimg** [comment]
Was an accident, my bad

Joaquim

**Comment by joaquimg** [comment]
Now it should be ok!

File: Xpress/versions/0.3.0/requires [hunk 3]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+MathProgBase 0.5 0.7
+Compat 0.8.6
```
Comment (tkelman) [inline Xpress/versions/0.3.0/requires]: not quite, still needs to be 0.18 minimum version for `@compat const Bounds{T<:Real}`

**Pull request merged**